src/main/java/com/cloudbees/jenkins/support/filter/ContentMappings.java

```diff
@@ -88,18 +88,27 @@ public static ContentMappings get() {
     private final Map<String, ContentMapping> mappings;
 
     private ContentMappings(@Nonnull XmlProxy proxy) {
-        stopWords = proxy.stopWords == null ? getDefaultStopWords() : proxy.stopWords;
-        mappings = proxy.stopWords == null
+        if (proxy.stopWords == null) {
+            stopWords = getDefaultStopWords();
+        } else {
+            stopWords = proxy.stopWords;
+            stopWords.add(Jenkins.VERSION);
+        }
+
+        mappings = proxy.mappings == null
                 ? new ConcurrentSkipListMap<>(COMPARATOR)
-                : proxy.mappings.stream().collect(toConcurrentMap(ContentMapping::getOriginal, Function.identity(), (a, b) -> {throw new IllegalArgumentException();}, () -> new ConcurrentSkipListMap<>(COMPARATOR)));
+                : proxy.mappings.stream()
+                    .filter(mapping -> !stopWords.contains(mapping.getOriginal()))
+                    .collect(toConcurrentMap(ContentMapping::getOriginal, Function.identity(), (a, b) -> {throw new IllegalArgumentException();}, () -> new ConcurrentSkipListMap<>(COMPARATOR)));
     }
 
     private static Set<String> getDefaultStopWords() {
         return new HashSet<>(Arrays.asList(
                 "jenkins", "node", "master", "computer",
                 "item", "label", "view", "all", "unknown",
                 "user", "anonymous", "authenticated",
-                "everyone", "system", "admin"
+                "everyone", "system", "admin",
+                Jenkins.VERSION
```

Comment:
Can you filter out Jenkins.VERSION from the stored stop words in `writeReplace` so that we don't slowly accumulate version numbers over time?

Reply:
Very good point, changed!